cmd/earthly/main.go

```diff
@@ -2606,9 +2606,15 @@ func (app *earthlyApp) actionBuild(c *cli.Context) error {
 			return errors.New("unable to use --ci flag in combination with --interactive flag")
 		}
 	}
-	if !termutil.IsTTY() && app.interactiveDebugging {
-		return errors.New("A tty-terminal must be present in order to the --interactive flag")
+	if app.interactiveDebugging {
+		if !termutil.IsTTY() {
+			return errors.New("A tty-terminal must be present in order to the --interactive flag")
+		}
+		if c.IsSet("buildkit-host") {
```

Comment:
Should this also apply if buildkit-host points to localhost?

Reply:
technically `--interactive` would work in that case, as would `127.0.0.1` or any other local host entry under `/etc/hosts`, but if that's the case I don't understand why someone would want to pass that value in rather than just leaving it empty.

Reply:
I generally agree that the use-case for setting buildkit host to localhost seems limited, but just calling out there are some places where we explicitly check if buildkit host is local (via IsLocal function https://github.com/earthly/earthly/blob/2888c9c530d994324c224654a40e895ed378aac5/buildkitd/buildkitd.go#L753). 

Maybe the notable thing is mTLS support locally requires you to set BUILDKIT_HOST to point to localhost, not sure how many people really want mTLS locally though.

Reply:
Maybe something like this in the meantime might be fine?
```
if c.IsSet("buildkit-host") && !buildkitd.IsLocal(app.buildkitHost) {
    ...
}
```

Reply:
ah great suggestion, I didn't realise we had a function to check that.